docs/gettingstarted.rst

```diff
@@ -23,7 +23,7 @@ Are you looking at installing and deploying REANA cluster locally on your laptop
 
    .. code-block:: console
 
-      $ minikube start --kubernetes-version="v1.9.4"
+      $ minikube start --kubernetes-version="v1.11.2"
```

Comment:
Works well on Arch Linux with `minikube start --kubernetes-version="v1.11.2" --vm-driver=kvm2`

Can you please also update Arch versions in `docs/userguide.rst` to read:

```console
pacman -Q | grep -iE '(docker|virtualbox|kube|qemu|libvirt)'
                                                                                                                            
docker 1:18.06.0-1
docker-compose 1.22.0-2
docker-machine 0.15.0-1
docker-machine-driver-kvm2 0.27.0-1
kubectl-bin 1.11.2-1
libvirt 4.6.0-3
minikube 0.28.2-1
python-docker 3.5.0-1
python-docker-pycreds 0.3.0-2
python-dockerpty 0.4.1-3
qemu 3.0.0-1
virtualbox 5.2.18-1
virtualbox-guest-iso 5.2.18-1
virtualbox-host-modules-arch 5.2.18-7
```

Reply:
@dinosk Perhaps you can try the bare `minikube start` without forcing k8s version to 1.11.2 on your super quick box to see what happens?

Reply:
For me when I start without specifying the Kubernetes version it defaults to 1.10 which would break R-J-Controller.

Reply:
updated and tested cluster creation with new configuration files.